package.json

```diff
@@ -38,7 +39,7 @@
     "test": "mocha",
     "lint": "eslint \"**/*.js\" Jakefile",
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha",
-    "doc": "rimraf out && jsdoc -c jsdoc.json lib/* docs/jsdoc/*",
+    "doc": "jake doc",
     "devdoc": "rimraf out && jsdoc -p -c jsdoc.json lib/* docs/jsdoc/*"
```

Comment:
mocha, eslint, etc. work as far as I know.

Parametrized jake task would make sense here. That would remove the need for the rimraf dep.